napari/_qt/utils.py

```diff
@@ -385,3 +386,18 @@ class Sentry(QObject):
 
     def alert(self, *_):
         self.alerted.emit()
+
+
+def qt_might_be_rich_text(text) -> bool:
+    """
+    Check if a text might be rich text in a cross-binding compatible way.
+    """
+    if qtpy.PYSIDE2:
+        from qtpy.QtGui import Qt as _Qt
+    else:
+        from qtpy.QtCore import Qt as _Qt
+
+    try:
+        return _Qt.mightBeRichText(text)
+    except Exception:
+        return bool(re.search("<[^\n]+>", text))
```

Comment:
I'm guessing is that goes into an event filter we want it to be as optimized  right ?
If that's the case would it be useful to extract   `pattern = re.compile('<[^\n]+>')`,  outside of the function to not have to recompile the regex each time ?

Reply:
note: 130ns vs 550ns roughly on my machine. so maybe not that critical unless we have a lots of tooltips

Reply:
Good idea, let me put the pattern outside.